src/main.rs

```diff
@@ -32,17 +32,16 @@ async fn index(data: Json<data::Notification>, bauth: bauth::BAuth, client: &Sta
         return Err(Status::Unauthorized);
     }
 
-    // Mapping grafana 'state' statuses^1 to the ntfy.sh emojis^23, so we have proper
-    // warning emoji on the alertings state and so on..  @Incomplete
-    //   ^1 - https://grafana.com/docs/grafana/latest/alerting/old-alerting/notifications/#webhook
+    // Mapping grafana 'status'^1 to the ntfy.sh emojis^23, so we have proper
+    // warning emoji on the alertings state and so on..
+    //   ^1 - https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier/
     //   ^2 - https://ntfy.sh/docs/publish/#tags-emojis
     //   ^3 - https://ntfy.sh/docs/emojis
     //
-    let tags_header = match data.state.as_str() {
-        "alerting" => format!("{}, {}", "warning", &data.state),
-        "ok" => format!("{}, {}", "white_check_mark", &data.state),
-        // TODO: There are more 'state' values to map.
-        _ => data.state.to_string(),
+    let tags_header = match data.status.as_str() {
+        "firing" => format!("{}, {}", "warning", &data.status),
+        "resolved" => format!("{}, {}", "white_check_mark", &data.status),
+        _ => data.status.to_string(),
```

Comment:
This looks like a breaking change, im assuming this won't work for older versions of grafana?
If so, can you update it to match both old and newer versions and just add a note so it can be deprecated at some point?

Reply:
This is a breaking change. The Status header was introduced in I believe 8.0, when they reworked their own alerting. The first mention of the alerting header tho, I found in the [8.3 docs](https://grafana.com/docs/grafana/v8.3/alerting/unified-alerting/contact-points/#webhook), which was released in February 2022. I don't know how far back you want to support. Keep in mind this was two months after the first working version of this project

Reply:
Oh, my grafana instance is newer than that (not by much). Whatever then, I'm gonna test with my setup and if it works, no need to bother.

Reply:
Thank you, if you merge this change, then the pictures in the README will have the wrong header values ('alerting' instead of 'firing'). Would that be a problem?

Reply:
Nope, doesn't work on my instance.

(Screenshots are fine)